runtime/conditions/merge_strategies.go

```diff
@@ -231,3 +233,10 @@ func getFirstCondition(g conditionGroups, priority []string) *localizedCondition
 		return &topGroup.conditions[0]
 	}
 }
+
+// WithGenerations instructs merge to consider the conditions with the given observed generations only.
+func WithGenerations() MergeOption {
```

Comment:
Wondering if something like `WithLatestGeneration()` would be more descriptive? Also, the comment seems to be outdated.

Reply:
Yeah, I think being explicit about it will be better.